portality/lib/dataobj.py

```diff
@@ -222,12 +222,16 @@ def sn(val):
 ############################################################
 
 ############################################################
-## The core data object which manages all the interactions
-## with the underlying data member variable
+# The core data object which manages all the interactions
+# with the underlying data member variable
+
 
 class DataObjException(Exception):
-    def __init__(self, msg, *args, **kwargs):
-        self.message = msg
+    def __init__(self, *args, **kwargs):
+        try:
+            self.message = args[0]
+        except IndexError:
+            self.message = ''
```

Comment:
I was trying to decide between 2 approaches here; I could have retained the signature and supply the message back into `args` with `args = (msg,) + args` but I decided since we already assume our message is the first positional argument this is equivalent and  cleaner.